independent-publisher-connectors/PrexView/apiDefinition.swagger.json

```diff
@@ -0,0 +1,128 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PrexView",
+    "description": "PrexView makes it simple to convert your data to different formats. Transform your data from XML or JSON to high quality, beautiful and readable documents in PDF, HTML, PNG or JPG.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.prexview.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/transform": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "$content-type": {
+                  "type": "string",
+                  "description": "The content type.",
+                  "title": "Content-Type"
+                },
+                "$content": {
+                  "type": "string",
+                  "description": "The file content.",
+                  "title": "Content"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create document",
+        "description": "Creates a document from the data based on a template in your selected format.",
+        "operationId": "TransformPost",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "xml": {
+                  "type": "string",
+                  "description": "Data to transform in JSON format.",
+                  "title": "XML"
+                },
+                "json": {
+                  "type": "string",
+                  "description": "Data to transform in JSON format.",
+                  "title": "JSON"
+                },
+                "output": {
+                  "type": "string",
+                  "description": "Type of document that will be created by PrexView service, it must be html, pdf, png or jpg.",
+                  "title": "Output",
+                  "enum": [
+                    "pdf",
+                    "html",
+                    "jpg",
+                    "png"
+                  ],
+                  "default": "pdf"
+                },
+                "template": {
+                  "type": "string",
+                  "description": "Template\u2019s name to use for the document creation, you can use dynamic values.",

```

Comment:
@troystaylor unicode symbols are not allowed, please replace them with appropriate symbols. There are a few of them in descriptions, search for `\u`.